common/src/main/java/com/wynntils/core/features/UserFeature.java

```diff
@@ -4,12 +4,13 @@
  */
 package com.wynntils.core.features;
 
+import com.wynntils.core.config.Configurable;
 import com.wynntils.core.config.properties.ConfigOption;
 
 /**
  * A feature that is enabled & disabled by the user.
  */
-public abstract class UserFeature extends Feature {
+public abstract class UserFeature extends Feature implements Configurable {
```

Comment:
Does it really make sense to have the base Feature class implement Configurable? That way even simple features without any "own" configuration needs to implement a dummy onUpdate. The functionality for turning features on and off can (and possibly should) be handled the "normal" config options. It's kind of a "meta-option".